ConsoleApp/Program.cs

```diff
@@ -79,13 +84,12 @@ static void Main(string[] args)
             }
 
             if (PressedKey == '1') {
-
+                CategorizeChoice();
```

Comment:
Da fehlt noch der Teil, wo man das Modell auswählt. Du kannst einfach meine Funktion `ClassificationChoice()` nutzen und vor deinen Code in CategorizeChoice() setzen